pmd-core/src/main/java/net/sourceforge/pmd/PMDException.java

```diff
@@ -13,6 +13,7 @@
  * @version $Revision$, $Date$
  * @since August 30, 2002
  */
+@Deprecated
```

Comment:
I'm not sure we should deprecate this.

I kinda like the idea, that we would provide all exceptions (if we throw any) with a common super type.
Of course, the exception should be not a checked exception like this one, but rather a runtime exception.

We maybe need additionally an internal exception that we would convert into such a public API exception.

But that's out of scope of this PR - that's the question of exception handling.